clients/src/main/java/org/apache/kafka/common/protocol/Protocol.java

```diff
@@ -1265,6 +1361,7 @@
                     "The partition")
             );
     public static final Schema OFFSET_FOR_LEADER_EPOCH_RESPONSE_V0 = new Schema(
+            newThrottleTimeField(),
```

Comment:
OFFSET_FOR_LEADER_EPOCH_RESPONSE is an inter broker request. So, we shouldn't add a throttle field.